include/deal.II/numerics/rtree.h

```diff
@@ -36,7 +36,7 @@ DEAL_II_NAMESPACE_OPEN
  * self-balancing spatial index (the R-tree) capable of storing various types of
  * values, using different balancing algorithms.
  *
- * From wikipedia:
+ * From Wikipedia (https://en.wikipedia.org/wiki/R-tree):
```

Comment:
We typically add links as hyperlinks (sorry that I wasn't explicit here):
```
From <a href="https://en.wikipedia.org/wiki/R-tree">Wikipedia</a>:
```

Reply:
There is no need. Doxygen supports markdown, which translates links to links explicitly, unless you want the more aesthetically pleasing link [Wikipedia](https://en.wikipedia.org/wiki/R-tree)